src/content/language/functions.md

```diff
@@ -266,6 +266,34 @@ assert(loudify('hello') == '!!! HELLO !!!');
 This example uses an anonymous function.
 More about those in the next section.
 
+## Function types
+
+You can specify the type of a function, which is known as a *function type*.
+A function type is obtained from a function declaration header by replacing 
+the function name by the word `Function`. Moreover, you are allowed to omit 
+the names of positional parameters, but the names of named parameters cannot 
+be omitted. For example:
+
+<?code-excerpt "misc/lib/language_tour/functions.dart (function-type)"?>
+```dart
+void greet(String name, {String greeting = 'Hello'}) =>
+    print('$greeting $name!');
+
+// Store `greet` in a variable and call it.
+void Function(String, {String greeting}) g = greet;
+g('Dash', greeting: 'Howdy');
+```
+
+:::note
+In Dart, functions are first-class objects,
+meaning they can be assigned to variables,
+passed as arguments, and returned from other functions.
+The [`typedef`][] keyword defines function types explicitly,
+which can be useful for clarity and reusability.
```

Comment:
Since `typedef` can be used for other type aliases and the keyword doesn't act alone in defining the type, perhaps let's restructure this to focus on what developers can use it for. How does this sound?

```suggestion

You can use a [`typdef`][] declaration to explicitly name function types,
which can be useful for clarity and reusability.
```